### ISSUE TITLE
Google Play Music not working

### ISSUE THREAD

**Issue by ianlewis** [web-scrobbler/web-scrobbler#1793]
Google Play Music is not working for me on Chrome 69.0.3497.120 (Official Build) (64-bit)
The extension says the website is supported for scrobbling but it doesn't recognize when a song is playing.

**Comment by jaccarmac** [comment]
Working for me. Could this have something to do with UI changes that are being previewed to your Google account? #1788 has similarly hard-to-reproduce failures, which makes me suspicious that the DOM is shifting dangerously underneath us.

**Comment by jaccarmac** [comment]
That's a long window for changing UI.

Do you have any other extensions installed that change how the page looks?

@inverse Could there be race conditions somewhere? I haven't taken a long look at the core of the scrobbler myself.

**Comment by HEdingfield** [comment]
Nope, I've disabled all extensions except the scrobbler and same issue. It works fine in Bandcamp, but no Google Music or Youtube. Works fine on my Windows machine, but continues to not function in Linux. Currently running 70.0.3538.102 (Official Build) (64-bit).

I have #1576 open for this, and saw mention of possibly needing a CSP in there. Maybe that's the issue?

**Comment by blanknamed** [comment]
(Firefox 64.0b9) [link ](https://music.youtube.com/playlist?list=OLAK5uy_l1Nlu9ig4gBGrSxy3Q0_H6OHScYozR6D0) where bug happens ![default](https://user-images.githubusercontent.com/9845271/48511487-84a8eb00-e85f-11e8-96ed-118c22e617b8.png)

**Comment by balamyt92** [comment]
Google Chrome 69.0.3497.100 Limux x64
Not working. Check debug log. Look session is not valid. Relogin last.fm in exstension settings. Works! It bad. Extension must do info user about this.

**Comment by alexesprit** [comment]
Is it still an issue?

**Comment by HEdingfield** [comment]
Yes, hasn't been working this whole time for Google Music and YouTube on Linux. Just verified with Chrome 74.0.3729.108 (Official Build) (64-bit).

**Comment by jaccarmac** [comment]
Working in 75 for me, though play state doesn't get unset when clearing queue. I don't really use GPM any more myself but if @HEdingfield can provide more details about their install I'll dig in and see if I can fix anything.

**Comment by jaccarmac** [comment]
@HEdingfield Note that Google Play Music, YouTube, and YouTube music all use different connectors. Please specify which one you are talking about when you mention bugs or the conversation becomes rather hard to follow (separate new issues for your failure case might be useful as well).

**Comment by alexesprit** [comment]
@jaccarmac they already posted the issue #1576.

Seems like the something, namely ExtensionsSettings policy, prevents injecting content scripts, and it applies to Google services only.

**Comment by alexesprit** [comment]
Closed as non-reproducable. If it does not work because of ExtensionsSettings policy, we cannot help.
